infrastructure/README.md

```diff
@@ -10,36 +44,52 @@ The `cdk.json` file tells the CDK Toolkit how to execute your app.
-If you want to access the Admin web you need to add /admin to the Distribution domain name.
-# Creating superuser
-## Prerequisites
+5. Once the implementation process has started through the console, you may be asked to confirm with a y/n, please confirm it.
+
+6. The url of the application will be displayed in the console
+
+### Creating superuser
+#### Prerequisites
 - You need to install AWS Session manager plugin (https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html).
 - You need to configure your AWS_PROFILE env variable.
 
-## Running the create_super_user.py script
+#### Running the create_super_user.py script
 To create a superuser run `python create_super_user.py`
 
 A superuser with the data configured in create_super_user.sh will be created.
 
 You can edit the file to create the superuser as desired. Take into account you need to edit it before delploying, since the script
 that gets executed is the one that is copied to the container.
-## Useful commands
+### Accessing the App
+Once the app is deployed you can access the web through the previously mentioned url. If you want to access the Admin web you need to add /admin to the Distribution domain name.
+
+# Destroy
+
+## Destroy deployment using pre-built image
+- If you want to remove all the resources deployed you can run: `docker run -e AWS_ACCESS_KEY_ID=... -e AWS_SECRET_ACCESS_KEY=... -e AWS_SESSION_TOKEN=... -e PROCESS=destroy -it video-headline-deploy`
+    - AWS_ACCESS_KEY_ID: AWS access key identifier.
+    - AWS_SECRET_ACCESS_KEY: AWS secret access key.
+    - AWS_SESSION_TOKEN: AWS session token (if required).
+
+   These variables can be found in AWS Command line or programmatic access.
+
+
+## Destroy deployment using local image
+
+- If you want to remove all the resources deployed you can run: `yarn cdk destroy '*' --profile [...]`
+
+# Useful commands
 -   `yarn run build` compile typescript to js
 -   `yarn run watch` watch for changes and compile
 -   `yarn run test` perform the jest unit tests
 -   `yarn cdk deploy` deploy this stack to your default AWS account/region
 -   `yarn cdk diff` compare deployed stack with current state
 -   `yarn cdk synth` emits the synthesized CloudFormation template
-## Configuring HTTPS
-If you want to configure https with a certificate you can put your certificate arn in `/lib/utils/aws.ts`. (https://github.com/qualabs/video-headline/blob/cca91831ef0130ea1fefa53084b949308d55c57e/infrastructure/lib/utils/aws.ts#L25).
-
-# Remove
-
-If you want to remove all the resources deployed you can run `yarn cdk destroy '*' --profile [...]`
+# Configuring HTTPS
+If you want to configure https with a certificate you can put your certificate arn in `/lib/utils/aws.ts`. (https://github.com/qualabs/video-headline/blob/cca91831ef0130ea1fefa53084b949308d55c57e/infrastructure/lib/utils/aws.ts#L25).
```

Comment:
What do you mean by "putting your certificate arn"?

Reply:
It was not necessary